core/src/main/scala/com/yahoo/maha/core/registry/Registry.scala

```diff
@@ -496,8 +506,10 @@ case class Registry private[registry](dimMap: Map[(String, Int), PublicDimension
       val maxDaysLookBack = JArray(toMaxDaysList(publicFact.maxDaysLookBack))
       val maxDaysWindow = JArray(toMaxDaysList(publicFact.maxDaysWindow))
 
+      val nameJson = if(!useRevisions) ("name" -> toJSON(publicFact.name)) else ("name,revision" -> toJSON(s"""${publicFact.name},${publicFact.revision}"""))
```

Comment:
we can not modify (backward incompatible change) existing domain json as it is already used by many.
you can expose new json without modifying existing one